File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex8.java
//Importando Scanner
import java.util.Scanner;

public class Ex8{
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars	
	float pesoK, pesoG;
	
	System.out.print("Peso Kg --> g\n\n");
	
	//Entrada
	System.out.print("Insira o peso em Kg: ");
	pesoK = input.nextFloat();
	
	//Processamento
	pesoG = pesoK * 1000;
	
	//Saída
	System.out.println("Peso em gramas: " + pesoG + " g");
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex8.c
#include <stdio.h>

main()
{
    //vars
    float pesoKilos, pesoGramas;

    printf("\t\t\tPeso Kilos --> Peso Gramas\n\n");

    //Entrada
    printf("Insira seu peso em KG: ");
    scanf("%f",&pesoKilos);

    //Processamento
    pesoGramas = pesoKilos * 1000;

    //Saída
    printf("Peso em gramas: %.0f", pesoGramas);

}
